plugins/mqtt/ua_mqtt_adapter.c

```diff
@@ -387,13 +478,13 @@ connectMqtt(UA_PubSubChannelDataMQTT* channelData){
 
     /* sync the first mqtt packets in the buffer to send connection request.
        After that yield must be called frequently to exchange mqtt messages. */
-    UA_StatusCode ret = yieldMqtt(channelData, 100);
-    if(ret != UA_STATUSCODE_GOOD){
-        freeTLS(channelData);
-        UA_free(channelData->connection);
-        UA_free(client);
-        return ret;
-    }
+    //UA_StatusCode ret = yieldMqtt(channelData, 100);
```

Comment:
Why is this commented out?
If not needed, remove entirely.

Reply:
removed